ground/gcs/src/plugins/config/configautotunewidget.h

```diff
@@ -27,28 +27,36 @@
 #ifndef CONFIGAUTOTUNE_H
 #define CONFIGAUTOTUNE_H
 
-#include "ui_autotune.h"
-#include "ui_autotuneproperties.h"
-#include "ui_autotunesliders.h"
-#include "ui_autotunefinalpage.h"
 #include "../uavobjectwidgetutils/configtaskwidget.h"
 #include "extensionsystem/pluginmanager.h"
 #include "uavobjectmanager.h"
 #include "uavobject.h"
 #include "actuatorsettings.h"
 #include "stabilizationsettings.h"
 #include "systemident.h"
-#include <QWidget>
+
+#include <QChart>
+#include <QLineSeries>
 #include <QTimer>
+#include <QWidget>
 #include <QWizardPage>
 #include <QtNetwork/QNetworkReply>
 
+QT_CHARTS_USE_NAMESPACE
+
+#include "ui_autotune.h"
+#include "ui_autotunebeginning.h"
+#include "ui_autotuneproperties.h"
+#include "ui_autotunesliders.h"
+#include "ui_autotunefinalpage.h"
 #include "configgadgetwidget.h"
 
 struct AutotunedValues
 {
     bool valid;
 
+    QByteArray *data;
```

Comment:
If it's a reference, it needs to be known at the moment of struct construction, doesn't it?  I don't think it's easy to structure the code that way.